docs/DeveloperGuide.md

```diff
@@ -257,42 +257,51 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​                                          | I want to …​                                     | So that I can…​                                                        |
+|----------|--------------------------------------------------|--------------------------------------------------|------------------------------------------------------------------------|
+| `* * *`  | producer                                         | see usage instructions                           | refer to instructions when I forget how to use the App                 |
+| `* * *`  | producer                                         | add a new person                                 | contact them to form a band                                            |
+| `* * *`  | producer                                         | delete a record for one musician                 | remove entries that I no longer need                                   |
+| `* * *`  | producer                                         | find a person by name                            | locate details of persons without having to go through the entire list |
+| `* *`    | producer                                         | tag a musician with an instrument they play      | know their potential role in a band                                    |
+| `* *`    | producer                                         | tag a musician with the genre they specialise in | find musicians suiting the song I am making                            |
+| `* *`    | producer                                         | edit a record for a musician                     | modify or update the information for a musician                        |
+| `* *`    | producer                                         | view all records of musicians I have stored      | easily see all the information I have                                  |
+| `* *`    | producer                                         | hide private contact details                     | minimize chance of someone else seeing them by accident                |
+| `*`      | producer who has worked with a musician before   | give a musician a rating                         | can keep track of my past experience with them                         |
+| `*`      | producer who is actively producing music         | check the availability of a musician             | I can avoid musicians with clashing schedules                          |
+| `*`      | producer with many musicians in the address book | sort persons by name                             | locate a person easily                                                 |
+
```

Comment:
More user stories to be added. I will add some according to our excel sheet after this skeleton is merged.